src/api/mapping.ts

```diff
@@ -0,0 +1,17 @@
+import express from 'express'
+import uuid4 from 'uuid/v4'
+const mappingRouter = express.Router()
+const list =[]
```

Comment:
If you store in memory array, then everytime the app restarts, all data is lost. Please edit this file and add `mappings` to be stored: https://github.com/garageScript/myProxy/blob/master/src/api/lib/data.ts